### ISSUE TITLE
0.13.0-rc is tagged as `latest` on npm

### ISSUE THREAD

**Issue by nevir** [facebook/react-native#3546]
```
>  npm dist-tag ls react-native
latest: 0.13.0-rc
```

Not sure if this was intended, but it means people will get `0.13.0-rc` when running `npm install react-native`.

_`npm publish` defaults to the latest tag, even for `-rc` builds :( Probably meant to `npm publish --tag 0.13.0-rc`?_

**Comment by ide** [comment]
Thanks for the note. It's now updated:
```
$ npm dist-tag ls react-native
latest: 0.12.0
```

**Comment by nevir** [comment]
:+1:

**Comment by cosmith** [comment]
Same for `0.14.0-rc`
```
npm dist-tag ls react-native
latest: 0.14.0-rc
```

**Comment by ide** [comment]
@cosmith thanks! should be fixed now (points at 0.13.2)
